notebooks/toy_model_with_strs.ipynb

```diff
@@ -0,0 +1,5411 @@
+{
```

Comment:
Is there a way to show the difference between this file and the original colab?
Or maybe you can change the colab directly(but don't merge)?

Reply:
For this, I uploaded the collab just for myself so I don't lose this version. We can delete this one eventually.